README.md

```diff
@@ -0,0 +1,345 @@
+
+Kabanero installs on OpenShift and integrates a modern DevOps toolchain and application stack hubs which enable developers to use runtimes and frameworks in pre-built container images called `Application stacks`.  
+
+Eclipse Codewind provides the ability to create application projects for these `Application Stacks` that your company has built, enabling developers to focus on their code and not infrastructure and Kubernetes.  Application deployments to Kubernetes occur via pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.    
+
+Eclipse Codewind provides the ability to create projects based on a variety of different template types.  These include IBM Cloud starters,Odo, and Appsody templates.  When used with Kabanero, you will only use Appsody templates for the Collections your company has enabled.  Today, there are collections for: IBM Cloud Starters, OpenShift Do (odo), Eclipse MicroProfile/Java EE, Springboot, Node.js, Node.js with Express, Node.js with Loopback.
+
+This guide describes developing a simple microservice using two different IDEs. Select the guide section based on your IDE of choice: 
+
+* [Developing with VS Code](#developing-with-vs-code)
+* [Developing with Eclipse](#developing-with-eclipse)
+
+## Developing with VS Code
+
+If you use Visual Studio Code (VS Code), you can use Codewind for VS Code to develop and debug your containerized projects from within VS Code using the workflow you already use today.
+
+### Prerequisite 
+
+Before you can develop a microservice with VS Code, you need to:
+
+* [Install Docker](https://docs.docker.com/install/) 
+    * **Note:** Make sure to install or upgrade to minimum Docker version 19.03. 
+* [Install VS Code](https://code.visualstudio.com/download)
+ 
+### Installing Codewind for VS Code
+
+The Codewind installation pulls the following images that form the Codewind backend:
+
+1. `eclipse/codewind-performance-amd64`
+2. `eclipse/codewind-pfe-amd64`
+
+The Codewind installation includes two parts:
+
+1. The VS Code extension installs when you install Codewind from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) and click `Install`. 
+    * Or, go to `View->Extensions`, search for Codewind, and click `Install`. 
+2. The Codewind backend containers install after you click `Install` when you are prompted. Clicking `Install` downloads the Codewind backend containers, ~1GB. 
+    * **Optional:** If you don’t click `Install` when the notification window first appears, you can access the notification again. Go to `View->Explorer`. Then click `Codewind` and hover the cursor over `Codewind` where there is a switch to turn Codewind on or off. Click the switch so that it is `On`. The notification window is displayed. 
+
+### Configuring Codewind with application stack hubs
```

Comment:
This is tricky! Kabanero Collections has gone, so a user following this guide would choose `Appsody stacks - appsodyhub` from the available template sources. Here is where you could also introduce configuring Codewind for an organizations customized application stacks. 

So perhaps the title needs to be something like "Configuring Codewind to use application stacks".  Then you could say something like, "Use the following procedure to select the Appsody templates. These templates include an Eclipse MicroProfile stack that you can use to follow this guide" .... then have your 5 steps, with the last one being Enable "Appsody templates".

Then under that, you can say, "If your organisation uses customized application stacks and you have been given a URL that points to an `index.json` file, you can add it to Codewind by repeating these steps and selecting `Add Template Sources` instead. Add the URL into the pop up window and save your changes.